src/test/java/uk/gov/hmcts/reform/em/orchestrator/stitching/StitchingServiceTest.java

```diff
@@ -119,7 +119,7 @@ public void doesNotAppendBinary() throws Exception {
         String processedString = (String) binarySuffixAdder.invoke(service, testString);
 
         Assert.assertEquals("testString/binary", processedString);
-}
+    }
```

Comment:
checkstyle being good about incorrect indentation. don't know why it didn't pick up on this when it was originally written